gluefactory/configs/COMMON_BENCHMARK_CONF.yaml

```diff
@@ -6,14 +6,14 @@ model:  # TODO: REPLACE MODEL WITH YOU MODEL CFG YOU WANT TO BENCHMARK
     max_num_keypoints: 1500  # setting for training, for eval: -1
     detection_threshold: 0.0 # set to <= 0 for training, for eval without fixed num of kp, do set to a good value
     nms_radius: 3
-    subpixel_refinement: True
```

Comment:
FYI: I just checked and both "true" or "True" workes in yaml files but we can of course keep it any way we want, so changes are fine